assets/js/googlesitekit/data/create-settings-store.js

```diff
@@ -278,23 +286,6 @@ export const createSettingsStore = (
 			return state.settings;
 		},
 
-		/**
-		 * Indicates whether the current settings have changed from what is saved.
-		 *
-		 * @since 1.6.0
-		 * @since 1.77.0 Added ability to filter settings using `keys` argument.
-		 * @since 1.129.0 Changed the approach to use validateHaveSettingsChanged callback.
-		 *
-		 * @param {Object}     state Data store's state.
-		 * @param {Array|null} keys  Settings keys to check; if not provided, all settings are checked.
-		 * @return {boolean} True if the settings have changed, false otherwise.
```

Comment:
Why remove the doc string for the selector? Let's keep it around (and update its documentation if needed, I'd think it would be since it's changed a bit now).

Reply:
I removed since we don;t have inline docs in other places for this kind of selectors, but yeah you are right, it makes more sense to keep it since we already have it